examples/flutter_gallery/android/fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,8 +1,8 @@
-(Flutter is alpha technology)
+(Flutter is beta technology)
 
-A gallery of widgets and behaviors, plus demos and vignettes, all built with Flutter. This is a demo of some of the features of Flutter, and some features like accessibility are not yet wired in.
+A gallery of widgets and behaviors, plus demos and vignettes, all built with Flutter.
 
-Flutter is a new project to help developers build high-performance, high-fidelity, mobile apps for iOS and Android from a single codebase.
+Flutter is a mobile development framework to help developers build high-performance, high-fidelity, mobile apps for iOS and Android from a single codebase.
```

Comment:
Please change this tag-line to:

Flutter is Google’s multi-platform mobile app SDK for crafting beautiful native experiences on iOS and Android in record time.

Reply:
Done